### ISSUE TITLE
Module will not install due to failed tests

### ISSUE THREAD

**Issue by Xliff** [briandfoy/raku-PrettyDump#7]
Output from installation attempt:

```
$ zef install PrettyDump
===> Searching for: PrettyDump
===> Testing: PrettyDump:ver<1.1.7>
# Module version is 1.1.7
# META6 version is 1.1.7
    # Failed test 'Dumping Match works'
    # at t/03.Match.t line 43
    # expected: 'Match=(
    #   :from(2),
    #   :hash(Map=()),
    #   :list(List=()),
    #   :made(Any),
    #   :orig("abcdef"),
    #   :pos(4),
    #   :to(4)
    # )'
    #      got: 'Match=(
    #   :made(Any),
    #   :hash(Map=()),
    #   :orig("abcdef"),
    #   :pos(4),
    #   :list(List=()),
    #   :from(2),
    #   :to(4)
    # )'
    # expected: 'Match=(
    #   :from(2),
    #   :hash(Map=()),
    #   :list(List=()),
    #   :made(Mu),
    #   :orig("abcdef"),
    #   :pos(4),
    #   :to(4)
    # )'
    #      got: 'Match=(
    #   :made(Any),
    #   :hash(Map=()),
    #   :orig("abcdef"),
    #   :pos(4),
    #   :list(List=()),
    #   :from(2),
    #   :to(4)
    # )'
    # Looks like you failed 1 test of 3
# Failed test 'Match'
# at t/03.Match.t line 36
# Looks like you failed 1 test of 5
===> Testing [FAIL]: PrettyDump:ver<1.1.7>
Aborting due to test failure: PrettyDump:ver<1.1.7> (use --force-test to override)
```

**Comment by Xliff** [comment]
I have submitted a PR to fix this issue as #8. Hope this helps!

**Comment by briandfoy** [comment]
This should be fix now. The tests pass for me. Thanks,
